### PR TITLE
fixed duplicated domainuid, added check to match events with domainuid

### PR DESCRIPTION
**Pull request by marinakog**
Jenkin Runs

https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/9562 ( Aux and Event Tests )
https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/9563 ( Aux and Event Tests )
https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/9565 ( Aux and Event Tests )
https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/9566 ( Aux and Event Tests )

https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/9550/ (entire run)
https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/9573 (entire run)
https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/9577 (entire run)

### CODE REVIEW

**Comment by marinakog** [comment]
https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/9502

**Review by anpanigr — APPROVED** [review]

**Review by vanajamukkara — APPROVED** [review]

**Comment by vanajamukkara** [comment]
pls run the entire parallel suite

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/ItKubernetesEvents.java [hunk 1]
```diff
@@ -741,7 +741,7 @@ void testK8SEventsStartStopWatchingNS(boolean enableClusterRoleBinding) {
     upgradeAndVerifyOperator(opNamespace, opParams);
 
     logger.info("verify NamespaceWatchingStopped event is logged in namespace {0}", domainNamespace2);
-    checkNamespaceWatchingStoppedEvent(opNamespace, domainNamespace2, null, "Normal", timestamp,
+    checkNamespaceWatchingStoppedEvent(opNamespace, domainNamespace2, domainUid, "Normal", timestamp,
```
Comment (sankarpn) [inline integration-tests/src/test/java/oracle/weblogic/kubernetes/ItKubernetesEvents.java:744]: domainUid should be null as before. NamespaceWatchingStopped event will not contin domainUID

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/ItKubernetesEvents.java [hunk 2]
```diff
@@ -807,7 +807,7 @@ void testK8SEventsStartStopWatchingNSWithLabelSelector(boolean enableClusterRole
         .execute();
 
     logger.info("verify NamespaceWatchingStopped event is logged in namespace {0}", domainNamespace3);
-    checkNamespaceWatchingStoppedEvent(opNamespace, domainNamespace3, null, "Normal", timestamp,
+    checkNamespaceWatchingStoppedEvent(opNamespace, domainNamespace3, domainUid, "Normal", timestamp,
```
Comment (sankarpn) [inline integration-tests/src/test/java/oracle/weblogic/kubernetes/ItKubernetesEvents.java:810]: domainUid should be null as before. NamespaceWatchingStopped event will not contin domainUID

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/ItKubernetesEvents.java [hunk 3]
```diff
@@ -892,7 +892,7 @@ void testK8SEventsStartStopWatchingNSWithRegExp(boolean enableClusterRoleBinding
     upgradeAndVerifyOperator(opNamespace, opParams);
 
     logger.info("verify NamespaceWatchingStopped event is logged in namespace {0}", domainNamespace5);
-    checkNamespaceWatchingStoppedEvent(opNamespace, domainNamespace5, null, "Normal", timestamp,
+    checkNamespaceWatchingStoppedEvent(opNamespace, domainNamespace5, domainUid, "Normal", timestamp,
```
Comment (sankarpn) [inline integration-tests/src/test/java/oracle/weblogic/kubernetes/ItKubernetesEvents.java:895]: domainUid should be null as before. NamespaceWatchingStopped event will not contin domainUID

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/ItKubernetesEvents.java [hunk 4]
```diff
@@ -923,7 +923,7 @@ void testK8SEventsStartStopWatchingNSWithDedicated() {
     checkEvent(opNamespace, opNamespace, null, NAMESPACE_WATCHING_STARTED, "Normal", timestamp);
 
     logger.info("verify NamespaceWatchingStopped event is logged in {0}", domainNamespace4);
-    checkNamespaceWatchingStoppedEvent(opNamespace, domainNamespace4, null, "Normal", timestamp, false);
+    checkNamespaceWatchingStoppedEvent(opNamespace, domainNamespace4, domainUid, "Normal", timestamp, false);
```
Comment (sankarpn) [inline integration-tests/src/test/java/oracle/weblogic/kubernetes/ItKubernetesEvents.java:926]: domainUid should be null as before. NamespaceWatchingStopped event will not contin domainUID

**Review by sankarpn — CHANGES_REQUESTED** [review]
Please revert back the ItKubernetesEvents tests are before.

**Review by sankarpn — COMMENTED** [review]
fix typo

**Review by sankarpn — APPROVED** [review]
lgtm

**Review by vanajamukkara — COMMENTED** [review]
There are failures in ItKubernetesEvents class in the results posted in the PR. I don't see those failures in nightly run https://build.weblogick8s.org:8443/job/wko-kind-nightly-parallel/830/. Are these intermittent? Is there a JIRA to track these failures?

**Comment by marinakog** [comment]
> There are failures in ItKubernetesEvents class in the results posted in the PR. I don't see those failures in nightly run https://build.weblogick8s.org:8443/job/wko-kind-nightly-parallel/830/. Are these intermittent? Is there a JIRA to track these failures?

Hi Vanaja, it was older runs, I have updated with newest results.

**Review by anpanigr — APPROVED** [review]
lgtm

**Review by vanajamukkara — APPROVED** [review]

**Pull request merged**